src/main/java/org/apache/commons/text/WordUtils.java

```diff
@@ -828,7 +828,7 @@ public static boolean isDelimiter(final int codePoint, final char[] delimiters)
      *
      * @param str         the string to be abbreviated. If null is passed, null is returned.
      *                    If the empty String is passed, the empty string is returned.
-     * @param lower       the lower limit.
+     * @param lower       the lower limit; < -1 value is treated as 0.
```

Comment:
Might be better ", negative treated as zero", as any negative value, not only `-1` is treated as 0